### ISSUE TITLE
complete version of pytorch implementation of ECNN

### ISSUE THREAD

**Issue by Hugo101** [paul-bd/DempsterShaferTheory#1]
Hi Paul,
thanks for your implementation of DS layer using PyTorch. I am using it and reimplement the ECNN. However, I always failed to train a good DS layer right now. (the loss will decrease to some values and does not reduce any more. the accuracy is always around 1% if using CIFAR100). So I am wondering whether you have the complete version of pytorch implementation or not. If does, could you share it?

Thank you!
Best

**Comment by bezirganyan** [comment]
Hi @Hugo101, I have exactly the same issue, the model does not learn at all, and the accuracy for CIFAR 10 is around 10% (so completely random). Have you found a solution to the issue? 

Will really appreciate any help, and @paul-bd if it is possible to share the full training code it would be a tremendous help. 

Thank you
